credmark/__main__.py

```diff
@@ -0,0 +1,2 @@
+from .cmf import credmark_dev
+credmark_dev.main()
```

Comment:
This file cannot be here. If it needs to be somewhere, credmark/cmf

Reply:
yes. removed __init__.py from this branch.